src/landscape/components/LandscapeNew.js

```diff
@@ -79,21 +82,33 @@ const LandscapeNew = () => {
     [dispatch]
   );
 
-  useEffect(() => {
-    if (success) {
-      navigate(`/landscapes/${landscape.slug}`);
-    }
-  }, [success, landscape, navigate, dispatch]);
+  const onUpdate = async updatedLandscape => {
+    setUpdatedLandscape(updatedLandscape);
+    return dispatch(saveLandscape(updatedLandscape)).then(data => {
+      const success = _.get('meta.requestStatus', data) === 'fulfilled';
+      if (success) {
+        navigate(`/landscapes/${landscape.slug}`);
+        trackEvent('Landscape created - Optional Steps', {
```

Comment:
@david-code David, what do you think about this, this feature of creating the landscape always after step 1 kinds of changes the logic for the create event in plusible tracking the map option, let me know what do you think

Reply:
In this case, it is actually updating not creating the landscape, but it is still something that the user is doing through the create landsacpe steps, so maybe it is still ok